temp_convert_a: 0.032622
temp_convert_b: -535.5